modules/feed/html/rollup.js

```diff
@@ -41,7 +41,8 @@ exports.create = function (api) {
     bumpFilter = returnTrue,
     displayFilter = returnTrue,
     updateStream, // override the stream used for realtime updates
-    waitFor = true
+    waitFor = true,
+    observableFilter // filter that can change during runtime
```

Comment:
I don't think this is necessary for the automatic refresh.

You could call `feedView.reload()` inside _public.js_.

Based on my original example:

```js
var filters = api.settings.obs.get('filters')

var feedView = api.feed.html.rollup(getStream, {
  rootFilter: function (item) {
    if (!filters()) return true
    return !(filters().following && getType(msg) === 'contact')
  },
  ...
})

// call reload whenever filters changes (equivalent to the refresh from inside rollup)
filters(feedView.reload)
```